pkg/webhook/webhook_test.go

```diff
@@ -57,6 +57,28 @@ func (suite *WebhookTestSuite) TestProcessWebhookPR() {
 	assert.NotNil(t, logrusEntry)
 }
 
+func (suite *WebhookTestSuite) TestProcessWebhookPRReview() {
+	t := suite.T()
+
+	webhook := &scm.ReviewHook{
+		Action: scm.ActionSubmitted,
+		Repo:   suite.TestRepo,
+		Review: scm.Review{
+			State: "APPROVED",
```

Comment:
quick question: does it also handle the "request changes" state?

Reply:
Yup! And dismissing a review.